social_core/backends/linkedin.py

```diff
@@ -48,14 +48,14 @@ class LinkedinOAuth2(BaseOAuth2):
     name = "linkedin-oauth2"
     AUTHORIZATION_URL = "https://www.linkedin.com/oauth/v2/authorization"
     ACCESS_TOKEN_URL = "https://www.linkedin.com/oauth/v2/accessToken"
-    USER_DETAILS_URL = "https://api.linkedin.com/v2/me?projection=({projection})"
+    USER_DETAILS_URL = "https://api.linkedin.com/v2/userinfo?projection=({projection})"
     USER_EMAILS_URL = (
         "https://api.linkedin.com/v2/emailAddress"
         "?q=members&projection=(elements*(handle~))"
     )
     ACCESS_TOKEN_METHOD = "POST"
     REDIRECT_STATE = False
-    DEFAULT_SCOPE = ["r_liteprofile"]
+    DEFAULT_SCOPE = ["email", "profile", "openid"]
```

Comment:
These scopes are for [OpenID Connect](https://learn.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/sign-in-with-linkedin-v2?context=linkedin%2Fconsumer%2Fcontext#authenticating-members), not OAuth2. We shouldn't make such changes in `LinkedinOAuth2` class

Reply:
`r_liteprofile` was not working, see https://github.com/python-social-auth/social-core/issues/922